diff --git a/apps/storybook/stories/InputGroup.stories.tsx b/apps/storybook/stories/InputGroup.stories.tsx
index 302eb5a8260..14e620782db 100644
--- a/apps/storybook/stories/InputGroup.stories.tsx
+++ b/apps/storybook/stories/InputGroup.stories.tsx
@@ -180,6 +180,8 @@ export const WithTokenizer: Story = {
           value={value}
           onChange={setValue}
           placeholder="Add fruits..."
+          description="Tokenizer's own description"
+          status={{type: 'error', message: "Tokenizer's own status"}}
           maxEntries={3}
           hasEntriesOnFocus
         />
